Tribler/Tools/boostchannel.py

```diff
@@ -0,0 +1,124 @@
+# Written by Egbert Bouman
+
+import sys
+import time
+import shutil
+import random
+import getopt
+import tempfile
+from traceback import print_exc
+
+from Tribler.Core.Session import Session
+from Tribler.Core.SessionConfig import SessionStartupConfig
+from Tribler.Policies.BoostingManager import BoostingManager, CreationDatePolicy, SeederRatioPolicy
+from Tribler.Policies.BoostingPolicy import RandomPolicy, CreationDatePolicy, SeederRatioPolicy
+from Tribler.community.channel.community import ChannelCommunity
+from Tribler.community.channel.preview import PreviewChannelCommunity
+from Tribler.community.allchannel.community import AllChannelCommunity
+
+
```

Comment:
I agree. @ardhipoetra maybe for now you can leave this file out as you suggested in a comment below this. After that, you can work on a twistd plugin perhaps.